examples/drush

```diff
@@ -1,13 +1,32 @@
 #!/usr/bin/env sh
```

Comment:
I'm inclined to rename this file to drush.wrapper for disambiguation with the root drush script. The docs can suggest a rename to `drush` when using it.